packages/amplication-server/src/core/git/git.provider.service.ts

```diff
@@ -54,7 +54,7 @@ export class GitProviderService {
     gitProviderArgs: GitProviderArgs,
     logger = this.logger
   ): Promise<GitClientService> {
-    return this.createGitClient(gitProviderArgs, logger);
+    return new GitClientService().create(gitProviderArgs, logger);
```

Comment:
@yuval-hazaz we use this service in git-pull-request service, but only one. Should we do the same there?
https://github.com/amplication/amplication/blob/master/packages/amplication-git-pull-request-service/src/pull-request/pull-request.service.ts#L46